docs/calicoctl/ipam.md

```diff
@@ -0,0 +1,86 @@
+
+# User guide for 'calicoctl ipam' commands
+
+This section describes the `calicoctl ipam` commands.
+
+This command allows an interface into Calico's IP address management to release 
+IP addresses from endpoints and view additional information about assigned IPs.
+
+Read the [calicoctl command line interface user guide](../calicoctl.md) for a full list of calicoctl commands.
+
+## Displaying the help text for 'calicoctl ipam' commands
+
+Run `calicoctl ipam --help` to display the following help menu for the 
+calicoctl ipam commands.
+
+```
+
+Usage:
+  calicoctl ipam release <IP>
+  calicoctl ipam info <IP>
+
+Description:
+  Manage Calico assigned IP addresses
+
+Warnings:
+  -  Releasing an in-use IP address can result in it being assigned to multiple
+     workloads.
+
+```
+
+## calicoctl ipam commands
+
+
+### calicoctl ipam release <IP>
+
+This command allows you to release an IP address that has been assigned to an 
+endpoint.  When an IP address is released, it becomes available for assignment 
+to any endpoint.
```

Comment:
"Note that this does *not* remove the IP from any existing endpoints that may be using it, so we generally recommend you use this command to clean up addresses from endpoints that were not cleanly removed from Calico."